src/main/java/com/blamejared/crafttweaker/impl_native/util/ExpandDamageSource.java

```diff
@@ -11,10 +11,38 @@
 import net.minecraft.entity.Entity;
 import net.minecraft.util.DamageSource;
 
+import java.util.HashMap;
+import java.util.Map;
+
 @ZenRegister
 @Document("vanilla/api/util/DamageSource")
 @NativeTypeRegistration(value = DamageSource.class, zenCodeName = "crafttweaker.api.util.DamageSource")
 public class ExpandDamageSource {
+    public static Map<String, DamageSource> PRE_REGISTERED_DAMAGE_SOURCES = new HashMap<>();
```

Comment:
@jaredlll08 What was that utility method called that you prefer for these `Util.make(...)` ?